### PR TITLE
[WWW-23] Updated Privacy Policy

### PR DESCRIPTION
**Pull request by eak12913**
This PR updates the privacy policy according to the document provided in JIRA ticket [WWW-23]

[WWW-23]: https://gruntwork.atlassian.net/browse/WWW-23

### CODE REVIEW

**Comment by oredavids** [comment]
@eak12913 It appears you committed a new auto-generated terms document. Jim mentioned leaving that out to me in the past except we made changes to the actual terms.

File: _data/privacy-policy.yml [hunk 1]
```diff
@@ -2,174 +2,240 @@
...
+        <p>This Privacy Policy applies to information we collect and use for our own purposes, including providing the Services and marketing our business. In these situations, Gruntwork is considered the “data controller” for purposes of the GDPR and similar data protection laws. This Privacy Policy does not apply to personal data Gruntwork processes on behalf of our customers in our capacity as a “data processor” (or a comparable role such as a “service provider” in certain jurisdictions). In these situations, we only process customer information in accordance with our customer’s instructions and our agreement with that customer.</p>
+        <p>By using any of Gruntwork’s Services, you confirm you have agreed to the Site Terms and/or Gruntwork Terms of Service (the “Terms”), to the extent permitted by law, and have read and understood this Privacy Policy.</p>
 
 - question: What information do we collect about you?
   items:
     - title:
       ans: |
-        <p>We collect or may collect certain information from and about users of the Gruntwork Services, including: </p>
+        <p><strong><i>Information You Give Us Directly</i></strong></p>
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:19]:
```suggestion
        <h3>Information You Give Us Directly</h3>
```

File: _data/privacy-policy.yml [hunk 2]
```diff
@@ -2,174 +2,240 @@
...
+        <p><strong><i>Information We Collect Automatically</i></strong></p>
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:34]:
```suggestion
        <h3>Information We Collect Automatically</h3>
```

File: _data/privacy-policy.yml [hunk 3]
```diff
@@ -2,174 +2,240 @@
...
+          <li>Location-related information, which we derive from things like your device’s IP address or your mobile device’s GPS signal;</li>
+          <li>Information about your visit to our Website or Services, such as the date and time of your visit, the information you searched for to find our Website, the website you came from, the website you visit when you leave us, and how you used or didn’t use our Services; </li>
+          <li>Information about your activity on our Services such as which modules you chose to view and which modules you chose to deploy;  </li>
+          <li>Errors, crash reports, and other information relating to technical problems on our Website or Services, including basic information about what happened internally leading up to the crash </li>
         </ul>
+        <p>Some of this information may be collected using cookies and similar technologies. To learn more about our use of cookies, please see the section entitled “We use cookies and related technologies” below. </p>
+        <p><strong><i>Information We Collect From Other Sources</i></strong></p>
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:44]:
```suggestion
        <h3>Information We Collect From Other Sources</h3>
```

File: _data/privacy-policy.yml [hunk 4]
```diff
@@ -2,174 +2,240 @@
...
-        <p>You also have the right to set guidelines for the retention and communication of your personal data after your death.</p>
-        <p>If you reside in a jurisdiction other than the European Economic Area, you may also have similar rights to the above. Please contact us if you would like to exercise one of these rights, and we will comply with any request to the extent required under applicable law.</p>
-        <p>In addition, where you believe that we have not complied with its obligation under this privacy policy or European law, you have the right to make a complaint to an EU Data Protection Authority, such as the UK Information Commissioner’s Office.</p>
-        <p>You can exercise any of these rights by <a href="/contact/" title="Contact Gruntwork">contacting us</a>.</p>
+        <p>Additional Disclosures for California Residents</p>
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:198]:
```suggestion
        <h3>Additional Disclosures for California Residents</h3>
```

File: _data/privacy-policy.yml [hunk 5]
```diff
@@ -2,174 +2,240 @@
...
+          <br />
+          <li>Inferences, including information about your interests and preferences. </li>
+          <br />
+          <li>Internet activity, including your interactions with the Services and what led you to Gruntwork. </li>
+        </ul>
+        <p><u>The Purposes for Our Collection:</u> We collect and use these categories of personal data for our business and commercial purposes described in the “How do we use your information” section above, including providing and improving the Services, maintaining the safety and security of the Services, and for advertising and marketing our business. </p>
+        <p><u>Third-Party Marketing and Your Rights (Opt-Out of “Sale”):</u> Gruntwork does not sell personal data to third parties for monetary value. However, the term “sale” is defined broadly in the CCPA. To the extent that “sale” under the CCPA is interpreted to include interest-based advertising or other data uses described in the “How do we use your information” section above, we will comply with applicable law with respect to those uses.</p>
+        <br />
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:217]:
```suggestion
```

File: _data/privacy-policy.yml [hunk 6]
```diff
@@ -2,174 +2,240 @@
...
+        <p><u>“Do Not Track” Signals:</u> We do not recognize or respond to any web browser-initiated “Do Not Track” signals. At present, no universally accepted standard exists on how companies should respond to “Do Not Track” signals. In the event a universally accepted standard is established, we will assess and provide an appropriate response to these signals. If you wish, you can configure most browsers to reject cookies or to notify you when you are sent a cookie, giving you a chance to decide whether or not to accept it. Consult the help section of your browser to learn more about how to do this. Please note that if you choose to remove or reject cookies, this could affect the availability and functionality of the Services. </p>
+        <br />
```
Comment (josh-padnick) [inline _data/privacy-policy.yml:219]:
```suggestion
```

File: pages/privacy-policy/index.html [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Privacy Policy
-modified_date: September 20, 2018
+modified_date: June 3, 2020
```
Comment (josh-padnick) [inline pages/privacy-policy/index.html:4]: This date format looks good, but can you make it consistent with https://gruntwork.io/cookie-policy/, which uses a leading `0` for the day in the date (i.e. "June 02, 2020" vs. "June 2, 2020").

**Comment by josh-padnick** [comment]
I wanted to get this live and since I'm the reviewer here I've committed updates and approved.

**Comment by eak12913** [comment]
Sounds good - if you want to release this to prod, just tag the next version.

**Pull request merged**